src/validation.cpp

```diff
@@ -2289,7 +2289,7 @@ void static UpdateTip(const CBlockIndex *pindexNew, const CChainParams& chainPar
             DoWarning(strWarning);
         }
     }
-    LogPrintf("%s: new best=%s height=%d version=0x%08x log2_work=%.8g tx=%lu date='%s' progress=%f cache=%.1fMiB(%utxo)", __func__, /* Continued */
+    LogPrintf("%s: new best=%s height=%d version=0x%08x log2_work=%.8g tx=%lu date='%s' progress=%f cache=%.1fMiB(%dutxo)", __func__, /* Continued */
```

Comment:
It's `unsigned int CCoinsViewCache::GetCacheSize()` so it should be `%uutxo`.

Reply:
@promag That doesn't matter for LogPrintf.

Reply:
Thanks! There are some `%u` in the though.